compiler/luci-interpreter/src/kernels/Softmax.cpp

```diff
@@ -60,5 +68,17 @@ void Softmax::evalFloat() const
                                  getTensorShape(output()), getTensorData<float>(output()));
 }
 
+template <typename T> void Softmax::evalQuantized() const
+{
+  tflite::SoftmaxParams params{};
+  // params.beta = _params.beta;
+  params.table = new float[256];
```

Comment:
I didn't find where this memory is deleted, looks like it leaks, could you check please?
If there are no delete you can do it for yourself after `Softmax` call, or allocate this array on stack, like
```suggestion
  float intermediate[256];
  params.table = intermediate;
```

Reply:
Oh, Okay, I will fix it.